contrib/crio-wipe/crio-wipe.bash

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+set -u
+set -e
+
+dir=${0%/*}
+source "$dir/lib.bash"
+source "$dir/config.bash"
+
+# process command variables
+if [ ! -z ${1+x} ]; then
```

Comment:
I recommend `[[` and `]]` everywhere unless there's a very specific reason (i.e. need to support ancient versions of bash or sh).  The single bracket causes a child bash process to spawn and execute `/usr/bin/test`.  The double-bracket is interpreted directly by bash in the same process running the script (so it's faster).

Reply:
fixed!